equinox/nn/_normalisation.py

```diff
@@ -267,3 +268,118 @@ def __call__(
             return out
         else:
             return out, state
+
+
+class RMSNorm(Module):
+    r"""
+    A simplified version of LayerNorm which rescales the inputs, but does not center
+    them. Optionally applies a learned reweighting of the transformed array afterward.
+
+    Given an input array $x$, this layer computes
+
+    $$\frac{x}{\sqrt{\varepsilon + \frac{1}{n}\Vert x \Vert^2_2}} \gamma + \beta$$
+
+    where $\Vert \cdot \Vert_2$ is the 2-norm, $n = \dim(x)$, and $\gamma$ is a
```

Comment:
I'd probably write it out explicitly, to avoid ambiguity whether "2-norm" is mean or sum.